docker.go

```diff
@@ -370,6 +371,9 @@ func commandDaemon(daemon Daemon) *exec.Cmd {
 	if daemon.Experimental {
 		args = append(args, "--experimental")
 	}
+	if daemon.IPTables {
```

Comment:
need to verify the flag, but if `iptables_off: true` then we should disable iptables, which would otherwise be enabled by default

```
if daemon.IPTablesOff {
  args = append(args, "--iptables=false")
}
```

Reply:
done